src/data/tx-chunks-data-source.ts

```
@@ -84,10 +110,22 @@
 
         const rangeStream = Readable.from(rangeResult.stream);
 
-        // Track metrics timing
-        const firstChunkTime = Date.now() - rangeStartTime;
+        let firstChunkTime = 0;
+
+        // Measure actual TTFB on first data event
+        rangeStream.once('data', () => {
+          firstChunkTime = Date.now() - rangeStartTime;
+          span.setAttribute('chunks.streaming.first_chunk_time_ms', firstChunkTime);
```

ESLint

[error] Replace `'chunks.streaming.first_chunk_time_ms',·firstChunkTime` with `⏎············'chunks.streaming.first_chunk_time_ms',⏎············firstChunkTime,⏎··········`

(prettier/prettier)